tests/rptest/services/redpanda_installer.py

```diff
@@ -348,21 +351,50 @@ def get_cached_data():
                 releases_resp = requests.get(url)
                 releases_resp.raise_for_status()
                 try:
-                    releases_json = releases_resp.json()
+                    releases_json_unfiltered = releases_resp.json()
                 except:
                     self._redpanda.logger.error(releases_resp.text)
                     raise
 
-                assert isinstance(releases_json, list)
+                assert isinstance(releases_json_unfiltered, list)
+
+                # filter out releases < 2020, when the current versioning scheme was introduced
+                releases_json = []
+                for release in releases_json_unfiltered:
+                    match = VERSION_RE.findall(release["tag_name"])
+                    if match:
+                        if int_tuple(match[0])[0:2] >= RELEASES_LOWER_BOUND:
+                            releases_json.append(release)
+                        else:
+                            self._redpanda.logger.info(
+                                f"Ignoring old release {release['tag_name']}")
+                    else:
+                        if release["tag_name"].startswith("release-"):
+                            # Tags like 'release-20.12.4' predate the modern Redpanda versioning scheme
+                            self._redpanda.logger.info(
+                                f"Ignoring legacy release {release['tag_name']}"
+                            )
+                        else:
+                            self._redpanda.logger.warn(
+                                f"Malformed release tag in repo: {release['tag_name']}"
```

Comment:
a malformed tag will cause early stop in `if len(releases_json) < len(releases_json_unfiltered):`

Reply:
It will, but past releases from https://github.com/redpanda-data/redpanda/releases/tag/v23.2.12 are good, so this will break only when a new release with a broken tag will be cut